### DISCUSSION TITLE
big world collection

### DISCUSSION THREAD

**Discussion by maxkleehu** [Melledy/Grasscutter#1434] [Q&A] [answered]
How do I get mushrooms and stuff to spawn. I see the commit for this feature and I have the files, but my world is showing nothing.

**Comment by zhaodice** [comment, marked as answer]
try disable bigworldscript in config file

**Comment by maxkleehu** [comment]
ty, i thought bigworldscript was responsible for chest spawning, the puzzles still showing so I guess not.
